commands/clean/content.php

```diff
@@ -20,23 +20,32 @@
 			}
 		}
 
-		// get the keys
-		$contentFields = array_keys($contentFields);
+		// get the keys and normalize to lowercase
+		$originalContentKeys = array_keys($contentFields);
+		$contentFieldKeys = array_map('strtolower', $originalContentKeys);
 
-		// get all field keys from blueprint
+		// get all field keys from blueprint and normalize to lowercase
 		$blueprintFields = array_keys($item->blueprint()->fields());
+		$blueprintFieldKeys = array_map('strtolower', $blueprintFields);
 
-		// get all field keys that are in $contentFields but not in $blueprintFields
-		$fieldsToBeDeleted = array_diff($contentFields, $blueprintFields);
+		// get all field keys that are in $contentFieldKeys but not in $blueprintFieldKeys
+		$fieldsToBeDeleted = array_diff($contentFieldKeys, $blueprintFieldKeys);
 
 		// update page only if there are any fields to be deleted
 		if (count($fieldsToBeDeleted) > 0) {
+			// create a mapping: lowercase => original field name
+			$lowercaseToOriginal = array_combine($contentFieldKeys, $originalContentKeys);
 
 			// flip keys and values and set new values to null
-			$data = array_map(fn ($value) => null, array_flip($fieldsToBeDeleted));
+			$data = array_map(
+				fn ($field) => null,
+				array_flip(array_intersect_key($lowercaseToOriginal, array_flip($fieldsToBeDeleted)))
```

Comment:
`Suggestion`: There is a lot going on in this line. Can we untangle it into separate steps that are easier to follow?